P/Pythia8/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Pythia8"
+version = v"8.244.0"
+
+# Collection of sources required to complete build
+sources = [
+    "http://home.thep.lu.se/~torbjorn/pythia8/pythia8244.tgz" =>
+    "e34880f999daf19cdd893a187123927ba77d1bf851e30f6ea9ec89591f4c92ca",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd pythia8244/
+./configure --prefix=${prefix} --enable-shared --enable-64bit
```

Comment:
Aren't you missing the configuration of the host (quite important for cross-compilations)?
```suggestion
./configure --prefix=${prefix} --host=${target} --enable-shared --enable-64bit
```

Reply:
Yeah, there's no --host on the configure script. I only get "unknown option --host" do you have a suggestion?

Reply:
Oh, I see: this is _not_ using autotools, yet they called a script with the same name, similar options, but not all of them.  They got the bonus points from section VII of https://archive.fosdem.org/2018/schedule/event/how_to_make_package_managers_cry/.  Great job!

Reply:
Yeah, particle physicists and code. If it compiles, it's probably OK...